projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -115,7 +111,6 @@ describe("Tabbing order - tests don't require user to be logged in", () => {
     });
 
     context('Accessories', () => {
-      // TODO: This test fails with the current navigation-ui implementation for unknown reasons.
```

Comment:
Same as above

Reply:
Yes, that particular e2e desktop test is now fixed.